zipline-gradle-plugin/build.gradle.kts

```diff
@@ -44,6 +45,14 @@ gradlePlugin {
   }
 }
 
+pluginManager.withPlugin("org.jetbrains.kotlin.js") {
+  tasks.withType(KotlinJsCompile::class.java).configureEach {
+    kotlinOptions {
+      freeCompilerArgs += listOf("-Xir-per-module")
```

Comment:
what does that do 🤔

Reply:
Turns on modular JS builds that emit a IR (Kotlin compiler intermediate representation) JS file per JS module instead of all of the JS in a single file.